flit_core/flit_core/config.py

```diff
@@ -551,6 +592,18 @@ def read_pep621_metadata(proj, path) -> LoadedConfig:
                 "file or text field required in [project.license] table"
             )
 
+    license_files = []
+    if 'license-files' in proj:
+        _check_type(proj, 'license-files', list)
+        globs = proj['license-files']
+        license_files = _license_files_from_globs(path.parent, globs)
+    else:
+        license_files = _license_files_from_globs(
+            path.parent, default_license_files_globs, use_defaults=True
+        )
```

Comment:
If `project.license.file` is specified, we can emit that as `License-File` metadata.

> If the specified license `file` is present in the source tree, build tools SHOULD use it to fill the License-File field in the core metadata, and MUST include the specified file as if it were specified in a license-file field.